src/components/DeployButton/DeployButton.tsx

```diff
@@ -52,10 +52,10 @@ export default class DeployButton extends React.PureComponent<Props> {
   }
 
   render() {
-    const { areEntitiesOutOfBoundaries, isLoading } = this.props
+    const { areEntitiesOutOfBoundaries, project, isLoading } = this.props
     const exceededMetric = this.getExceededMetric()
     const didExceedMetrics = exceededMetric !== '' || areEntitiesOutOfBoundaries
-    const isButtonDisabled = isLoading || didExceedMetrics
+    const isButtonDisabled = isLoading || didExceedMetrics || !project.thumbnail
```

Comment:
Disable the deploy button in the EditorPage when the Scene doesn't have a valid thumbnail